### PR TITLE
Show in agent table the disconnection reason

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
The `status details` column is added in the agent table 
 
### Issues Resolved
- #5051

### Evidence

image

### Test

Navigate to agents and view the column `status details` in the table

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Tostti — CHANGES_REQUESTED** [review]

File: docker/imposter/agents/agents.json [hunk 1]
```diff
@@ -32,8 +32,12 @@
         "manager": "wazuh-manager-master-0",
         "node_name": "master",
         "lastKeepAlive": "9999-12-31T23:59:59Z",
+        "version": "Wazuh v4.5.0",
+        "group_config_status": "not synced",
+        "status_code": 0
```
Comment (Tostti) [inline docker/imposter/agents/agents.json:37]: The property `status_code` is duplicated

**Review by Tostti — CHANGES_REQUESTED** [review]

File: docker/imposter/agents/agents.json (hunk 1 above)
Comment (Desvelao) [reply docker/imposter/agents/agents.json]: Solved in https://github.com/wazuh/wazuh-kibana-app/pull/5680/commits/18a0e95ff5bdaad7b44021be745ca77c701f0bea.

**Review by Tostti — APPROVED** [review]
Testing can confirm that the PR works as expected
image

To make it work, I had to manually delete the previous localstorage. Confirmed that that issue is being fixed in https://github.com/wazuh/wazuh-kibana-app/issues/4562. 

All the comments were resolved

TEST:✔️
CR:✔️

LGTM

File: plugins/main/public/components/agents/agent_status.tsx
[the remark this answers is not in the document]
Comment (Tostti) [reply plugins/main/public/components/agents/agent_status.tsx:1]: Issue fixed in https://github.com/wazuh/wazuh-kibana-app/pull/5680/commits/850d8b2f5e9071aec752bddea9cc16f777a01dde

**Comment by yenienserrano** [comment]
it was decided to change the column to a tooltip with the information

image

**Review by jbiset — APPROVED** [review]
CR: :green_circle: 

TEST: :green_circle: 

## TEST EVIDENCE

Evidence_1

Evidence_2

Evidence_3

Evidence_4

**Review by chantal-kelm — APPROVED** [review]
Captura de pantalla 2023-08-02 a la(s) 11 41 19

**Pull request merged**